firestore-tos-extension/POSTINSTALL.md

```diff
@@ -0,0 +1,83 @@
+### See it in action
+
+You can test out this extension right away!
+
+1. Go to your [Authentication dashboard](https://console.firebase.google.com/project/${param:PROJECT_ID}/authentication/users) in the Firebase console.
+
+1. Click **Add User** to add a test user, then copy the test user's UID to your clipboard.
+
+2. Access the user through any of the authetication methods.
+
+```js
+    const user = await getAuth().getUser(uid);
+```
+
+3. Provide the user / userId into one of the available functions.
+
+### Using the extension
+
+All ackowledgement documents are managed in the Cloud Firestore database.
+
+These can be found in ${param:COLLECTION_PATH}.
+
+This will also contain two pre-defined subcollections:
+
+- acknowledgements
+- terms
+
+The extension will provide `4` new functions for managing ackowledgement for users.
+
+**Accept Terms**
+
+  ```js
+    await acceptTermsFn.call({},
+      { tosId }, { auth: { uid: user.uid } }
```

Comment:
The UID should be sent (and handled) with the context. If the user is signed in with Firebase, these will be automatically sent.

Reply:
Updated to simply pass the required data in the examples. <v9 and v9 examples also added as part of the update.